distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSecurity.java

```diff
@@ -480,6 +486,214 @@ public void testBasicRestAccess() throws Exception {
         getSecurityPolicy().waitForGuestAuthReady(url);
     }
 
+    @Test(expected = NoHttpResponseException.class)
+    public void testBasicAuthOverHttp() throws Exception {
+        String url = SERVICE_ROOT.getUrl() + "/catalog/query?q=*&src=local";
+
+        //try basic auth over http
+        try {
+            configureRestForBasic(SDK_SOAP_CONTEXT);
+            waitForSecurityHandlers(url);
+
+            given().auth()
+                    .basic("admin", "admin")
+                    .when()
+                    .get(url.replace("https://", "http://"));
+        } finally {
+            configureRestForGuest(SDK_SOAP_CONTEXT);
+            getSecurityPolicy().waitForGuestAuthReady(url);
+        }
+    }
+
+    @Test
+    public void testTLSv11() throws Exception {
+        String url = SERVICE_ROOT.getUrl() + "/catalog/query?q=*&src=local";
+
+        SSLContext sslContext = SSLContext.getInstance("TLSv1.1");
+        sslContext.init(null, null, null);
+        SSLSocketFactory sslSocketFactory = new SSLSocketFactory(sslContext);
+
+        RestAssuredConfig config = new RestAssuredConfig();
+        config.sslConfig(SSLConfig.sslConfig()
+                .sslSocketFactory(sslSocketFactory));
+
+        checkAuthenticationRequest(config, url);
+    }
+
+    @Test
+    public void testTLSv12() throws Exception {
+        String url = SERVICE_ROOT.getUrl() + "/catalog/query?q=*&src=local";
+
+        SSLContext sslContext = SSLContext.getInstance("TLSv1.2");
+        sslContext.init(null, null, null);
+        SSLSocketFactory sslSocketFactory = new SSLSocketFactory(sslContext);
+
+        RestAssuredConfig config = new RestAssuredConfig();
+        config.sslConfig(SSLConfig.sslConfig()
+                .sslSocketFactory(sslSocketFactory));
+
+        checkAuthenticationRequest(config, url);
+    }
+
+    @Test
+    public void testTLSv1() throws Exception {
+        String httpsProtocols = System.getProperties()
+                .getProperty("https.protocols");
+        String url = SERVICE_ROOT.getUrl() + "/catalog/query?q=*&src=local";
+
+        try {
+            System.setProperty("https.protocols", "TLSv1");
+
+            SSLContext sslContext = SSLContext.getInstance("TLSv1");
+            sslContext.init(null, null, null);
+            SSLSocketFactory sslSocketFactory = new SSLSocketFactory(sslContext);
+
+            RestAssuredConfig config = new RestAssuredConfig();
+            config.sslConfig(SSLConfig.sslConfig()
+                    .sslSocketFactory(sslSocketFactory));
+
+            configureRestForBasic(SDK_SOAP_CONTEXT);
+            waitForSecurityHandlers(url);
+
+            given().config(config)
+                    .auth()
+                    .basic("admin", "admin")
+                    .when()
+                    .get(url)
+                    .then()
+                    .assertThat()
+                    .statusCode(401);
```

Comment:
I haven't seen any traffic on this comment in a while. It's been over week. 
@AzGoalie @pklinef @stustison where do you stand on this?